clocker-cli/src/main/java/com/scalar/clocker/repository/TimestampRepository.java

```diff
@@ -0,0 +1,112 @@
+package com.scalar.clocker.repository;
+
+
+import com.scalar.clocker.exception.ClockerCliException;
+import com.scalar.clocker.exception.RepositoryException;
+import com.scalar.clocker.model.Timestamp;
+import com.scalar.db.api.DistributedTransaction;
+import com.scalar.db.api.DistributedTransactionManager;
+import com.scalar.db.api.Get;
+import com.scalar.db.api.Put;
+import com.scalar.db.api.Result;
+import com.scalar.db.api.Scan;
+import com.scalar.db.exception.transaction.TransactionException;
+import com.scalar.db.io.BigIntValue;
+import com.scalar.db.io.Key;
+import com.scalar.db.io.TextValue;
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.ZoneOffset;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+public class TimestampRepository extends ScalarDbReadOnlyTransactionRepository<Timestamp> {
+
+  public static final String NAMESPACE = "clocker";
+  public static final String TABLE_NAME = "timestamp";
+  private final DistributedTransactionManager db;
+
+  public TimestampRepository(DistributedTransactionManager db) {
+    this.db = db;
+  }
+
+  public Timestamp create(LocalDateTime localDateTime, String user, String location) {
+    try {
+      DistributedTransaction tx = db.start();
+      Get get = createGet(localDateTime.toLocalDate(), localDateTime.toLocalTime(), user);
+      getAndThrowsIfAlreadyExist(tx, get, Timestamp.class);
+
+      Put put =
+          new Put(
+              createPk(localDateTime.toLocalDate()),
+              createCk(localDateTime.toLocalTime(), user))
+              .withValues(
+                  Arrays.asList(
+                      new TextValue(Timestamp.LOCATION, location),
+                      new BigIntValue(
+                          Timestamp.CREATED_AT,
+                          localDateTime.toInstant(ZoneOffset.UTC).toEpochMilli())))
+              .forNamespace(NAMESPACE)
+              .forTable(TABLE_NAME);
+      tx.put(put);
+      tx.commit();
```

Comment:
`commit()` throws  CommitConflictException and it should be caught for the same reason.